setup.cfg

```diff
@@ -20,6 +20,8 @@ ignore =
     E203, E266, E501, W503,
     # Ignored to conform to PEP257
     D203, D212, D213, D214, D215, D404, D405, D406, D407, D408, D409, D410, D411, D413, D415, D416, D417,
+    # Ignore documentation
+    D100, D101, D102, D103, D104, D105, D107, DAR101, DAR102, D106,
```

Comment:
What is this? Definitely we must NOT ignore documentation checks. 

This explains why flake8 didn't complain about missing docstrings, but I am not as easy as that program to shut down!

Reply:
As said, this will be removed in later commits within the PR, as we come closer to move away from draft PR